### ISSUE TITLE
XLNet fine-tuning speed (Multi-label classification)

### ISSUE THREAD

**Issue by dbswndigh** [huggingface/transformers#2197]
## ❓ Questions & Help

Hi!

I'm wondering whether my speed of fine-tuning is normal.

Training is taking **2~3hours per epoch**.

I am using [fast-bert](https://github.com/kaushaltrivedi/fast-bert) to train multi-label classifier, with
- model = xlnet-base-cased
- max sequence length = 512 tokens
- mem states using = no
- gpu = V100 * 4
- training data amount = 110000 ~ 120000
- validation data amount = 30000 ~ 40000
- evaluation = per each epoch, per every 100 steps
- apex, f16 used

Thank you so much!
